### DISCUSSION TITLE
AMD GPU VRAM allocation error

### DISCUSSION THREAD

**Discussion by DSN2002** [comfyanonymous/ComfyUI#1899] [Q&A] [answered]
Hello everyone, my GPU is AMD's RX5600XT, which has 6G VRAM, but ComfyUI can only recognize 1G VRAM. When outputting images or using Controlnet, it will prompt that VRAM is insufficient. Is there any way to make ComfyUI use all 6G VRAM?——from Google Translate

![屏幕截图 2023-11-04 164701](https://github.com/comfyanonymous/ComfyUI/assets/89450486/ca8b7478-88cd-4eff-a8f5-fd0b1d77b615)

**Comment by lord-lethris** [comment, marked as answer]
This is the normal for us AMD Based users I'm afraid, until Rocm windows support improves with pytorch, or MS DirectML get their finger out and improve that.

**Reply by DSN2002** [reply]
Thank you for your reply.
